src/verification/android.rs

```diff
@@ -35,7 +38,7 @@ enum VerifierStatus {
 const AUTH_TYPE: &str = "RSA";
 
 /// A TLS certificate verifier that utilizes the Android platform verifier.
-#[derive(Default)]
+#[derive(Default, Debug)]
```

Comment:
Tiny nit: alphabetic?

(There is at least one other instance.)

Reply:
Fixed throughout :+1: